### ISSUE TITLE
Resign/New Opponent/Abort Stopped working

### ISSUE THREAD

**Issue by oafysgbk** [veloce/lichobile#1131]
**I'm submitting a ...**  (keep the relevant one):
- bug report

# Bug report

**Current behavior:**

I am running the Lichess app on iOS (iPhone 8 Plus). I noticed today (7 Feb 2020) that the app had auto-updated to the new version. I can no longer resign during games. After a game ends, New Opponent no longer works, and I have to go back to the home screen to start a new game. While waiting for a new game, Abort Game no longer works. I don't typically try to rematch after games but I believe that button is also broken. I have already tried rebooting my phone and reinstalling the app - no luck.

**Expected behavior:**

Pretty self-explanatory - these buttons should work.

**Steps to reproduce:**

N/A - buttons just don't work.

**Environment information**

iPhone 8 Plus
12.1
7.0.0

**Other information:**

# Feature request

**Comment by Oleg2tor** [comment]
I have same issue

**Comment by bwhrsc** [comment]
Same issue. iPhone 7. Have had the problem ever since auto update recently (can't see ver number in app).

**Comment by veloce** [comment]
I cannot reproduce the issue. Does the issue persist if you delete the app (and all it's data) and re-install it?

**Comment by oafysgbk** [comment]
> I cannot reproduce the issue. Does the issue persist if you delete the app (and all it's data) and re-install it?

Yes. I’ve just tried again. Deleted the app, restarted my phone, and reinstalled the app. Issues persist. All post game buttons do not work: analysis board, share, and new opponent. During the game I could not resign. Give time does not work. Basically nothing works.

**Comment by oafysgbk** [comment]
Update/workaround: I updated my phone to the latest iOS and now the app appears to be working correct. I don’t know if I just got lucky or if this will work for everyone.

**Comment by ZeChArtiahSaher** [comment]
Same, iphone 6s ios 12.1.4

**Comment by dm-z** [comment]
iPad Air 2, iOS 11.4.1 same problem, it is now impossible to play, bunch of stuff doesn't work.

**Comment by kierankelly19** [comment]
I am having the same issue on iOS 12.1.2. Looks like with the update it broke something for iOS 12 users. It also shows up on the settings screen in the analysis board so I can’t toggle local evaluation anymore.

**Comment by bgradin** [comment]
+1 for the issue being related to iOS 12. I updated to iOS 13 and the in game menu buttons are usable again.

**Comment by veloce** [comment]
thanks for the info. I'll try to reproduce specifically on ios 12.
